src/index.ts

```diff
@@ -45,3 +49,20 @@ export const buildServicesFromConfig = async (): Promise<Services> => {
 
     return {ceramic}
 }
+
+export default class IntegrationTestEnvironment extends NodeEnvironment {
+    async setup() {
+        await super.setup();
+
+        const services = await buildServicesFromConfig();
+        this.global.services = services;
+    }
+
+    async teardown() {
+        // @ts-ignore
```

Comment:
I'm not sure why this is needed, but without it I get an error during build saying:
```
src/index.ts:62:17 - error TS2339: Property 'ceramic' does not exist on type '{}'.

62         const { ceramic } = this.global.services;    
```

Reply:
You'll need to declare it so TS know what is in the global object, ex: https://github.com/ceramicstudio/js-idx/blob/0d8135f3e7000da4214614b84430a598345be70b/test/integration.test.ts#L11-L13

Reply:
Unfortunately I still get an error:
```
src/index.ts:30:35 - error TS2339: Property 'close' does not exist on type 'unknown'.

30         await this.global.ceramic.close();
```

Reply:
Are you importing the `CeramicApi` type as in https://github.com/ceramicstudio/js-idx/blob/0d8135f3e7000da4214614b84430a598345be70b/test/integration.test.ts#L5
If so, might be that the package is not installed or can't be resolved somehow.

Reply:
Well, I have this line: https://github.com/3box/ceramic-integration-tests/pull/4/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80L1

package.json already included `@ceramicnetwork/core`, which I would have assumed brought in `common` transitively, but I just added an explicit dependency on `common` to be safe, but that didn't change anything

Reply:
I guess that link didn't work, but line 1 of `index.ts` is `import { CeramicApi, IpfsApi } from '@ceramicnetwork/common';`

Reply:
Looks like you're not explicitly importing the type as in https://github.com/ceramicstudio/js-idx/blob/0d8135f3e7000da4214614b84430a598345be70b/test/integration.test.ts#L5 though, no?

Reply:
What's missing?  I have `import { CeramicApi, IpfsApi } from '@ceramicnetwork/common';`.  Is that not the exact same as what you linked?